policy_packs/aws/ssm/enforce_install_packages_on_ec2_instances_using_ssm_package_installer/README.md

```diff
@@ -0,0 +1,205 @@
+---
+categories: ["security"]
+primary_category: "security"
+---
+
+# Enforce Install Packages on AWS EC2 Instances Using AWS SSM Package Installer
```

Comment:
Should we use `AWS` twice, or just with EC2? Using it each time seems easier to stay consistent but potentially redundant. Do we have any other examples of multiple services in a title?

Reply:
The dir name should be consistent too, e.g., if we do decide to keep in `AWS`, the dir name should be `on_aws_ec2_` as well (and vice versa if we only include it once).

Reply:
> Should we use AWS twice, or just with EC2? Using it each time seems easier to stay consistent but potentially redundant. Do we have any other examples of multiple services in a title?
I don't think we do. Feels redundant though. Will remove the second `AWS`.